.github/workflows/github-actions-test.yml

```diff
@@ -0,0 +1,27 @@
+name: test
+on : [pull_request, push]
+jobs:
+  mcat-action-test:
+    runs-on: macos-latest
+    steps:
+      - run: echo "The name of your branch is ${{ github.ref }} and your repository is ${{ github.repository }}."
+      - name: Checkout repository code
+        uses: actions/checkout@v2
+
+      - name: Setup python
+        uses: actions/setup-python@v2
+        with: 
+          python-version: '3.7'
+          architecture: 'x64'
+      
+      - name: Install python packages
+        run: |
+          python -m pip install --upgrade pip
+          pip install virtualenv
+          pip install --upgrade pip
```

Comment:
upgrade pip here is a duplicate of line 19 + you no longer need to install virtualenv.

Reply:
Done!!